internal/frontend/handler.go

```diff
@@ -110,8 +110,8 @@ func (handler *StaticHandler) serveFile(w http.ResponseWriter, r *http.Request,
 		} else {
 			w.Header().Set("Referrer-Policy", "strict-origin-when-cross-origin")
 
-			w.Header().Set("Content-Security-Policy", "default-src 'self'; img-src * data: ; "+
-				";connect-src 'self' https://*.auth0.com ;font-src 'self' data: "+
+			w.Header().Set("Content-Security-Policy", "default-src 'self' https://*.userpilot.io; img-src * data: ; "+
+				";connect-src 'self' https://*.auth0.com https://*.userpilot.io wss://*.userpilot.io ;font-src 'self' data: "+
```

Comment:
arent we including the package directly from package.json? or does it still load JS code remotely?